File: backendworker/NHSOnline.Backend.Worker/GpSystems/Suppliers/Tpp/Models/Appointments/ListSlotsReply.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using System.Xml.Serialization;

namespace NHSOnline.Backend.Worker.GpSystems.Suppliers.Tpp.Models.Appointments
{
    [Serializable]
    public class ListSlotsReply
    {
        [XmlAttribute("patientId")]
        public string PatientId { get; set; }

        [XmlAttribute("onlineUserId")]
        public string OnlineUserId { get; set; }

        [XmlAttribute("bookableDays")]
        public int BookableDays { get; set; }

        [XmlElement("Session", typeof(Session))]
        public List<Session> Sessions { get; set; }

        [XmlAttribute("uuid")]
        public Guid Uuid { get; set; }
    }
}

File: bddtests/src/main/kotlin/mocking/tpp/models/ListSlotsReply.kt
package mocking.tpp.models

import javax.xml.bind.annotation.XmlAccessType
import javax.xml.bind.annotation.XmlAccessorType
import javax.xml.bind.annotation.XmlAttribute
import javax.xml.bind.annotation.XmlRootElement
import javax.xml.bind.annotation.XmlElement

@XmlRootElement(name = "ListSlotsReply")
@XmlAccessorType(XmlAccessType.FIELD)
data class ListSlotsReply (
    @XmlAttribute var patientId: String = "default patientId",
    @XmlAttribute var onlineUserId: String = "default onlineUserId",
    @XmlAttribute var uuid: String = "default uuid",
    @XmlAttribute var bookableDays: String = "defualt bookableDays",
    @XmlElement var Session: MutableCollection<Session> = mutableListOf()
)